pydeequ/suggestions.py

```diff
@@ -183,6 +185,12 @@ class CategoricalRangeRule(_RulesObject):
 
     @property
     def rule_jvm(self):
+        if IS_DEEQU_V2:
+            # DISCLAIMER: this is a workaround for using the default category sorter
```

Comment:
Could you shed some light why we have to pass a parameter in V2?

Reply:
Definitely! This new argument was added [here](https://github.com/awslabs/deequ/commit/5ecefaa2f2efd380e220a652be3b4eec4495cab2) on Deequ 2.x. If we try to instantiate this object without passing this parameter, py4j raises a 
```
Constructor com.amazon.deequ.suggestions.rules.CategoricalRangeRule([]) does not exist
```
exception (as reported [here](https://github.com/awslabs/python-deequ/issues/70)). The "workaround" here lives in the fact that we are forcing this parameter to the default Scala value, so we don't need to expose it to the Python API.

If you have any ideas on how we could officially support passing the `categoricalSorter` parameter from Python, I'll gladly update it! 😃

Reply:
Hmm I wonder how PySpark (and its Scala counterpart) generally handle those optional parameter stuff.. but it's ok for now as long as this solves #70. 

Could you open a new issue for us to revisit this?